openstudiocore/resources/energyplus/ProposedEnergy+.idd

```diff
@@ -7006,29 +6972,29 @@ WindowProperty:ShadingControl,
        \key OnIfHighOutdoorAirTempAndHighHorizontalSolar
        \key OnIfHighZoneAirTempAndHighSolarOnWindow
        \key OnIfHighZoneAirTempAndHighHorizontalSolar
-  A5 , \field Schedule Name
+  A6 , \field Schedule Name
        \type object-list
        \object-list ScheduleNames
        \note Required if Shading Control Is Scheduled = Yes.
        \note If schedule value = 1, shading control is active, i.e., shading can take place only
        \note if the control test passes. If schedule value = 0, shading is off whether or not
        \note the control test passes. Schedule Name is required if Shading Control Is Scheduled = Yes.
        \note If Schedule Name is not specified, shading control is assumed to be active at all times.
-  N1 , \field Setpoint
-       \units BasedOnField A4
+  N2 , \field Setpoint
+       \units W/m2, W or deg C
```

Comment:
Running `IddFixture.IddFactory_Units` test, this leads to `C++ exception with description "/home/julien/Software/Others/OpenStudio/openstudiocore/src/utilities/units/Unit.cpp@654 : W/m^2, W or deg K is not a UnitString, per openstudio::isUnit." thrown in the test body.`

I found this comment [here](https://github.com/jmarrec/OpenStudio/blob/develop/openstudiocore/src/utilities/idd/IddField.cpp#L628) which made me laugh

Reply:
Reverted in https://github.com/jmarrec/OpenStudio/pull/2/commits/e10a63e5898ce4f0e45fa6177f57d1c6c4e08480